src/components/tx/SponsoredBy/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Stack, SvgIcon, Typography } from '@mui/material'
+import GasStationIcon from '@/public/images/common/gas-station.svg'
+import useRemainingRelays from '@/hooks/useRemainingRelays'
+import css from './styles.module.css'
+
+const MAX_HOUR_RELAYS = 5
```

Comment:
This value is also declared in the dashboard widget. Let's export it from one place and use that everywhere else.

Reply:
I export it from `<SponsoredBy />` now